validation/policies/io/konveyor/forklift/vmware/name.rego

```diff
@@ -22,8 +22,8 @@ concerns[flag] {
     valid_vm
     not valid_vm_name
     flag := {
-        "category": "Critical",
+        "category": "Warning",
         "label": "Invalid VM Name",
-        "assessment": "The VM name must comply with the DNS subdomain name format defined in RFC 1123. The name can contain lowercase letters (a-z), numbers (0-9), and hyphens (-), up to a maximum of 64 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (.), or special characters."
+        "assessment": "assessment": "The VM name must comply with the DNS subdomain name format defined in RFC 1123. The name can contain lowercase letters (a-z), numbers (0-9), and hyphens (-), up to a maximum of 64 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (.), or special characters. The VM name will be automatically changed during the migration to meet the RFC convention."
```

Comment:
```suggestion
        "assessment": "assessment": "The VM name must comply with the DNS subdomain name format defined in RFC 1123. The name can contain lowercase letters (a-z), numbers (0-9), and hyphens (-), up to a maximum of 64 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (.), or special characters. The VM will be automatically renamed during the migration to meet the RFC convention."
```
and why do we need the duplicate `assessment`?

Reply:
One is for ovirt and the second one for VMware, from what I saw in the validation structure the validation that's executed per plan are depending on the provider, and since both of them need this they have two separate validation for it.

Reply:
I meant the `"assessment": "assessment":` part - and I see you changed it